### ISSUE TITLE
additional_args

### ISSUE THREAD

**Issue by toni-k8s** [kijai/ComfyUI-FluxTrainer#177]
I added --max_data_loader_n_workers 2 in additional_args but it still didn't work. Or where should I add it? Please answer. Thank you.

<img width="469" height="733" alt="Image" src="https://github.com/user-attachments/assets/b8852703-5a53-4828-8380-0cc70bb3a3d3" />
Namespace(console_log_level=None, console_log_file=None, console_log_simple=False, v2=False, v_parameterization=False, pretrained_model_name_or_path='/root/autodl-tmp/ComfyUI/models/unet/Flux/flux1-dev-fp8.safetensors', tokenizer_cache_dir=None, train_data_dir=None, cache_info=False, shuffle_caption=False, caption_separator=',', caption_extension='.caption', caption_extention=None, keep_tokens=0, keep_tokens_separator='', secondary_separator=None, enable_wildcard=False, caption_prefix=None, caption_suffix=None, color_aug=False, flip_aug=False, face_crop_aug_range=None, random_crop=False, debug_dataset=False, resolution=None, cache_latents=True, vae_batch_size=1, cache_latents_to_disk=True, skip_cache_check=False, enable_bucket=False, min_bucket_reso=256, max_bucket_reso=1024, bucket_reso_steps=64, bucket_no_upscale=False, token_warmup_min=1, token_warmup_step=0.0, alpha_mask=False, dataset_class=None, caption_dropout_rate=0.0, caption_dropout_every_n_epochs=0, caption_tag_dropout_rate=0.0, reg_data_dir=None, in_json=None, dataset_repeats=1, output_dir='/root/autodl-tmp/output/ovo', output_name='daxiang_rank32_bf16', huggingface_repo_id=None, huggingface_repo_type=None, huggingface_path_in_repo=None, huggingface_token=None, huggingface_repo_visibility=None, save_state_to_huggingface=False, resume_from_huggingface=False, async_upload=False, save_precision='bf16', save_every_n_epochs=None, save_every_n_steps=None, save_n_epoch_ratio=None, save_last_n_epochs=None, save_last_n_epochs_state=None, save_last_n_steps=None, save_last_n_steps_state=None, save_state=False, save_state_on_train_end=False, resume=None, train_batch_size=1, max_token_length=None, mem_eff_attn=True, torch_compile=False, dynamo_backend='inductor', xformers=True, sdpa=False, vae=None, max_train_steps=1000, max_train_epochs=None, max_data_loader_n_workers=0, persistent_data_loader_workers=False, seed=42, gradient_checkpointing=True, gradient_accumulation_steps=1, mixed_precision='bf16', full_fp16=False, full_bf16=True, fp8_base=True, fp8_dtype='e4m3', ddp_timeout=None, ddp_gradient_as_bucket_view=False, ddp_static_graph=False, clip_skip=None, logging_dir=None, log_with=None, log_prefix=None, log_tracker_name=None, wandb_run_name=None, log_tracker_config=None, wandb_api_key=None, log_config=False, noise_offset=None, noise_offset_random_strength=False, multires_noise_iterations=None, ip_noise_gamma=None, ip_noise_gamma_random_strength=False, multires_noise_discount=0.3, adaptive_noise_scale=None, zero_terminal_snr=False, min_timestep=None, max_timestep=None, loss_type='l2', huber_schedule='snr', huber_c=0.1, huber_scale=1.0, lowram=False, highvram=False, sample_every_n_steps=None, sample_at_first=False, sample_every_n_epochs=None, sample_prompts=["daxiang. This is an exquisite bronze elephant ornament. The elephant has a vivid shape, with a bright blue-green mat on its back, decorated with golden patterns, which looks gorgeous and exotic. The elephant's trunk is raised high and elegantly bent into an S shape, with a round spherical container of the same bronze color hanging at the end, which seems to be openable. The metal texture and color matching of the entire ornament are very exquisite, and the details are properly handled, showing a unique artistic beauty."], sample_sampler='ddim', config_file=None, output_config=False, metadata_title=None, metadata_author=None, metadata_description=None, metadata_license=None, metadata_tags=None, prior_loss_weight=1.0, conditioning_data_dir=None, masked_loss=False, deepspeed=False, zero_stage=2, offload_optimizer_device=None, offload_optimizer_nvme_path=None, offload_param_device=None, offload_param_nvme_path=None, zero3_init_flag=False, zero3_save_16bit_model=False, fp16_master_weights_and_gradients=False, optimizer_type='adamw8bit', use_8bit_adam=False, use_lion_optimizer=False, learning_rate=4.000000000000002e-05, max_grad_norm=0.5, optimizer_args=[], lr_scheduler_type='', lr_scheduler_args=None, lr_scheduler='cosine', lr_warmup_steps=50, lr_decay_steps=0, lr_scheduler_num_cycles=1, lr_scheduler_power=1.0, fused_backward_pass=False, lr_scheduler_timescale=None, lr_scheduler_min_lr_ratio=None, dataset_config='[[datasets]]\nresolution = [ 512, 768,]\nbatch_size = 3\nenable_bucket = true\nbucket_no_upscale = false\nmin_bucket_reso = 512\nmax_bucket_reso = 768\n[[datasets.subsets]]\nimage_dir = "/root/autodl-tmp/output10"\nclass_tokens = "daxiang"\nnum_repeats = 10\n\n\n[general]\nshuffle_caption = false\ncaption_extension = ".txt"\nkeep_tokens_separator = "|||"\ncaption_dropout_rate = 0.0\ncolor_aug = false\nflip_aug = false\n', cache_text_encoder_outputs=True, cache_text_encoder_outputs_to_disk=True, text_encoder_batch_size=None, disable_mmap_load_safetensors=False, weighting_scheme='logit_normal', logit_mean=0.0, logit_std=1.0, mode_scale=1.29, blocks_to_swap=0, min_snr_gamma=5.0, scale_v_pred_loss_like_noise_pred=False, v_pred_like_loss=None, debiased_estimation_loss=False, weighted_captions=False, cpu_offload_checkpointing=True, no_metadata=False, save_model_as='safetensors', unet_lr=None, text_encoder_lr=[0.0, 0.0], fp8_base_unet=False, network_weights=None, network_module='.networks.lora_flux', network_dim=32, network_alpha=16.0, network_dropout=None, network_args=[], network_train_unet_only=False, network_train_text_encoder_only=False, training_comment=None, dim_from_weights=False, scale_weight_norms=None, base_weights=None, base_weights_multiplier=None, no_half_vae=False, skip_until_initial_step=False, initial_epoch=None, initial_step=None, clip_l='/root/autodl-tmp/ComfyUI/models/clip/clip_l.safetensors', t5xxl='/root/autodl-tmp/ComfyUI/models/clip/t5xxl_fp8_e4m3fn.safetensors', ae='/root/autodl-tmp/ComfyUI/models/vae/ae.safetensors', t5xxl_max_token_length=512, apply_t5_attn_mask=True, guidance_scale=1.0, timestep_sampling='flux_shift', sigmoid_scale=1.0, model_prediction_type='raw', discrete_flow_shift=3.1582000000000003, bypass_flux_guidance=False, num_cpu_threads_per_process=1, spda=False)

**Comment by toni-k8s** [comment]
-- max_data_loader_n_workers  8
